api/tests/scripts/beneficiary/update_gender_and_married_name_from_ubble_test.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+import re
+from unittest.mock import patch
+
+from dateutil.relativedelta import relativedelta
+import pytest
+
+from pcapi.core.fraud import factories as fraud_factories
+from pcapi.core.fraud import models as fraud_models
+from pcapi.core.users import factories as users_factories
+from pcapi.core.users import models as users_models
+from pcapi.scripts.beneficiary.update_gender_and_married_name_from_ubble import (
+    update_gender_and_married_name_from_ubble,
+)
+
+from tests.connectors.beneficiaries.ubble_fixtures import UBBLE_IDENTIFICATION_RESPONSE
+
+
+@pytest.mark.usefixtures("db_session")
+class UpdateGenderAndMarriedNameFromUbbleTest:
+    def setup(self):
+        # Not yet beneficiary
+        self.user_not_beneficiary = users_factories.UserFactory(
+            dateOfBirth=datetime.datetime.now() - relativedelta(year=18, month=4),
+        )
+        fraud_factories.BeneficiaryFraudCheckFactory(
+            user=self.user_not_beneficiary,
+            type=fraud_models.FraudCheckType.UBBLE,
+        )
+
+        # Beneficiary with gender
+        self.user_beneficiary_up_to_date = users_factories.BeneficiaryGrant18Factory(gender=users_models.GenderEnum.F)
+
+        # Beneficiary with no gender or married_name
+        self.user_beneficiary_to_update = users_factories.BeneficiaryGrant18Factory(gender=None, married_name=None)
```

Comment:
Ça a son importance ici, tu veux or ou nor ?
```suggestion
        # Beneficiary with no gender nor married_name
        self.user_beneficiary_to_update = users_factories.BeneficiaryGrant18Factory(gender=None, married_name=None)
```

Reply:
Nor effectivement